plugins/module_utils/foreman_helper.py

```diff
@@ -357,20 +358,31 @@ def _delete_entity(self, resource, current_entity, params):
         payload = {'id': current_entity['id']}
         if params:
             payload.update(params)
-        self._resource_action(resource, 'destroy', payload)
+        self.resource_action(resource, 'destroy', payload)
         return True, None
 
-    def _resource_action(self, resource, action, params):
+    def resource_action(self, resource, action, params, **kwargs):
         resource_payload = self.foremanapi.resource(resource).action(action).prepare_params(params)
         try:
             result = None
             if not self.check_mode:
-                result = self.foremanapi.resource(resource).call(action, resource_payload)
+                result = self.foremanapi.resource(resource).call(action, resource_payload, **kwargs)
```

Comment:
I guess, this is to pass `file=`?

Maybe the same reasoning as in
https://github.com/Apipie/apypie/pull/42#issuecomment-518642495
applies.

Reply:
yes.

Reply:
almost, I also pass `skip_validation`, because apypie can't validate the file :D